packages/flutter/lib/src/gestures/events.dart

```diff
@@ -334,8 +334,9 @@ class PointerAddedEvent extends PointerEvent {
     int device = 0,
     Offset position = Offset.zero,
     bool obscured = false,
-    double pressureMin = 1.0,
-    double pressureMax = 1.0,
+    double pressure = 1.0,
+    double pressureMin = 0.0,
+    double pressureMax = 6.0,
```

Comment:
what is 6.0? This seems like a very arbitrary number.

Reply:
This default range seems very wrong to me. If you create a default PointerAdded event, it should have the most conservative values. 0-6 doesn't make sense e.g. for a mouse (which should be 1.0-1.0).

Reply:
It's 6 because it's close to 6.66 which is the maximum value that apple uses for the pressure in the event itself. What value would you suggest? A mouse doesn't have any pressure by the way?

Reply:
The default min-max should be a conservative default. If the engine sends a range from the device platform, the override should come from the engine.